### PR TITLE
Add support for maximum amount of agents that should work on a supertask

### PR DESCRIPTION
**Pull request by cc-nfi**
This is a continuation on #764. The maxAgents configuration for a super task limits the maximum cumulative amount of agents working on on the subtasks.

maxagents

### CODE REVIEW

**Comment by zyronix** [comment]
Fixes #769

**Review by zyronix — COMMENTED** [review]
`curl http://127.0.0.1:8080/api/user.php -H "Content-Type: application/json" -X POST -d '{"section": "task", "request": "runPretask", "name": "API test", "hashlistId":1, "pretaskId":1, "crackerVersionId": 1, "accessKey": ""}'`
Fails, because a taskwrapper object is not updated.

**Review by zyronix — COMMENTED** [review]
image

The bulkcreate currently is missing setting the maxAgents. I don't think this is an issue, but just something I noticed.

File: doc/user-api/sections/task.tex
[the remark this answers is not in the document]
Comment (zyronix) [reply doc/user-api/sections/task.tex:1]: No, only the name and pretasks are required for creating a supertask in the old api.

**Comment by s3inlc** [comment]
There seems to be an issue when updating the Max Agents value on the Supertasks page. After saving it redirects to the pretasks page and stays there instead of going back to the supertasks page, i.e. the `?super=true` does not seem to have the desired effect.

**Comment by zyronix** [comment]
Fixed the broken redirect.

**Review by s3inlc — APPROVED** [review]

**Pull request merged**